src/provider.tsx

```diff
@@ -1,15 +1,16 @@
 import React from "react";
-import { Provider as KeyHandlerProvider } from "./focused-stack/context";
+import { Provider as KeyHandlerProvider} from "./focused-stack/context";
 import { FocusGroup } from "./focus-group";
 
 interface OwnProps {
   children: React.ReactNode;
   timeOut?: number; // Number of miliseconds to wait before timing our a melody
 }
 
-export function Provider({ children, timeOut = 2000 }: OwnProps) {
+export function Provider({ children, timeOut=2000 }: OwnProps) {
+
   return (
-    <KeyHandlerProvider timeOut={timeOut}>
+    <KeyHandlerProvider timeout={timeOut}>
```

Comment:
I think we still might want to rename this as well, right?

```suggestion
    <KeyHandlerProvider timeout={timeout}>
```